code/game/objects/items/melee/baton.dm

```diff
@@ -519,9 +519,12 @@
 
 /obj/item/melee/baton/security/attack_self(mob/user)
 	if(cell?.charge >= cell_hit_cost)
-		active = !active
-		balloon_alert(user, "turned [active ? "on" : "off"]")
-		playsound(src, SFX_SPARKS, 75, TRUE, -1)
+		if(SSsecurity_level.get_current_level_as_number() >= SEC_LEVEL_RED)
+			active = !active
+			balloon_alert(user, "turned [active ? "on" : "off"]")
+			playsound(src, SFX_SPARKS, 75, TRUE, -1)
+		else
+			to_chat(user, span_warning("You cannot use your baton in a non emergency situation"))
```

Comment:
```suggestion
			balloon_alert(user, "This is not an emergency situation! You find yourself cripplingly unable to use your stun baton, since you have a steadfast and righteous respect for the rules and regulations of this station!")
```

reads better imo

Reply:
please stop making troll suggestions

Reply:
and balloon alerts are not recommended for self interactions

Reply:
Also San balloon alerts are not supposed to be verbose since they disappear in a few seconds. Generally their text is short and sweet:  `"cant do that!"` `"need a wrench!"` etc.